Dockerfile

```diff
@@ -9,3 +9,5 @@ COPY .generator.yaml .generator.yaml
 COPY templates templates
 
 CMD ["sh", "-c", "java -jar /opt/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -c .generator.yaml -i ${OPENAPI_SPEC} -g go -o isp --skip-validate-spec --git-user-id=istreamlabs --git-repo-id=go-sdk"]
+# use this for isp-slate api
+# CMD ["sh", "-c", "java -jar /opt/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -c .generator.yaml -i ${OPENAPI_SPEC} -g go -o isp-slate --skip-validate-spec --git-user-id=istreamlabs --git-repo-id=go-sdk"]
```

Comment:
Can we have a separate dockerfile for slates so that we don't need to uncomment to generate the slates sdk?